test/Makefile.mk

```diff
@@ -5,11 +5,12 @@
 
 EXTRA_DIST += runtests.py
 
-SUBDIRS = test/dtpools
-
 LDADD = libyaksa.la test/dtpools/libdtpools.la
+
```

Comment:
Unrelated white space cleanup.  Should be in a separate patch.

Reply:
fixed.